### PR TITLE
[MIG] stock_picking_type_shipping_policy_group_by: Migration to 14.0

### PR DESCRIPTION
**Pull request by phuctranfxvn**
depends on

- [ ] https://github.com/OCA/stock-logistics-workflow/pull/874

### CODE REVIEW

**Review by simahawk — CHANGES_REQUESTED** [review]
missing old commits?

**Comment by sebalix** [comment]
@simahawk no the module was already in 14.0 with `installable: False` flag

**Review by sebalix — APPROVED** [review]
LG, thanks, it remains the conflict on `test-requirements.txt`

File: test-requirements.txt [hunk 1]
```diff
@@ -1 +1,5 @@
 freezegun
+
```
Comment (simahawk) [inline test-requirements.txt]: to clean up

**Review by mmequignon — APPROVED** [review]
LG, just the conflict to fix

**Comment by hailangvn** [comment]
Thanks for reviewing. I will work on it.

**Comment by hailangvn** [comment]
Good day @mmequignon, @sebalix, @simahawk. This PR has been updated and rebased.

**Review by simahawk — APPROVED** [review]

**Pull request merged**